test/functional/tests/volumes/test_volume_partitions.py

```diff
@@ -0,0 +1,182 @@
+        core_dev = TestRun.disks['core']
+        core_size = []
+        for i in range(cores_number):
+            core_size.append(Size(1, Unit.GibiByte))
+        core_dev.create_partitions(core_size, partition_table)
+
+    with TestRun.step("Create filesystem on core devices."):
+        for i in range(cores_number):
+            core_dev.partitions[i].create_filesystem(filesystem)
+
+    with TestRun.step("Mount core devices and create test files."):
+        counter = 1  # There is one partition (4th) which is omitted while creating cache
+        files = []   # and continuous numbering is needed here.
+        for core in core_dev.partitions:
+            mount_path = mount_point + str(counter)
+            core.mount(mount_path)
+            test_file_path = f"{mount_path}/test_file"
+            files.append(fs_utils.create_test_file(test_file_path))
+            counter += 1
+
+    with TestRun.step("Check md5 sums of test files."):
+        test_files_md5sums = []
+        for file in files:
+            test_files_md5sums.append(file.md5sum())
+
+    with TestRun.step("Unmount core devices."):
+        for core in core_dev.partitions:
+            core.unmount()
+
+    with TestRun.step(f"Start cache."):
+        cache = casadm.start_cache(cache_dev, cache_mode, force=True)
+
+    with TestRun.step("Add cores to cache."):
+        cores = []
+        for i in range(cores_number):
+            cores.append(cache.add_core(core_dev.partitions[i]))
+
+    with TestRun.step("Mount core devices."):
+        for core in cores:
+            mount_path = mount_point + str(core.core_id)
+            core.mount(mount_path)
+
+    with TestRun.step("Check again md5 sums of test files."):
+        test_files_md5sums_new = []
+        for file in files:
+            test_files_md5sums_new.append(file.md5sum())
+
+    with TestRun.step("Unmount core devices."):
+        for core in cores:
+            core.unmount()
+
+    with TestRun.step("Stop cache."):
+        cache.stop()
+
+    with TestRun.step("Compare md5 sums of test files."):
+        for i in range(cores_number):
+            if test_files_md5sums[i] != test_files_md5sums_new[i]:
+                TestRun.fail("Md5 sums are different.")
+
+
+@pytest.mark.parametrize("cache_mode", CacheMode)
+@pytest.mark.parametrize("partition_table", PartitionTable)
+@pytest.mark.parametrize("filesystem", Filesystem)
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+def test_partition_create_cas(partition_table, filesystem, cache_mode):
+    """
+        title: Volume test for preserving partition table on core device after stopping OpenCAS.
+        description: |
+          Validation of the ability of CAS to preserve partition table on core device
+          after stopping cache.
+        pass_criteria:
+          - Md5 sums shall on partitions be identical before and after running cache.
```

Comment:
Md5 sums on partitions shall be identical before and after [something that makes sense here].